presto-main/src/main/java/io/prestosql/sql/planner/RelationPlanner.java

```diff
@@ -192,6 +194,40 @@ private RelationPlan addRowFilters(Table node, RelationPlan plan)
         return new RelationPlan(planBuilder.getRoot(), plan.getScope(), plan.getFieldMappings());
     }
 
+    private RelationPlan addColumnMasks(Table table, RelationPlan plan)
+    {
+        Map<String, List<Expression>> columnMasks = analysis.getColumnMasks(table);
```

Comment:
Shouldn't `getColumnMasks()` return `Multimap`?

Reply:
I wanted to be explicit about the ordering of the expressions, so a Map with a List seemed more appropriate.